src/main/api/2/studio-api-2.yaml

```diff
@@ -3488,6 +3488,52 @@ paths:
         '403':
           $ref: '#/components/responses/Forbidden'
 
+  /configuration/content-type/preview_image:
+    get:
+      tags:
+        - configuration
+      summary: Get preview image of a given content-type
+      operationId: getContentTypePreviewImage
+      parameters:
+        - name: siteId
+          in: query
+          description: site ID
+          required: true
+          schema:
+            type: string
+        - name: contentType
+          in: query
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: 'OK'
+          content:
+            '*/*':
+              schema:
+                type: string
```

Comment:
Is this the correct type?

Reply:
@sumerjabri We only want to respond the binary image, right? If it is true, the type here is correct. The response is a string in binary format (same as in https://app.swaggerhub.com/apis/craftercms/studio/4.0.1.0#/plugin/getPluginFile )